addons/azuredisk-csi-driver/0.4.x/azuredisk-csi-driver-2.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: azuredisk-csi-driver
+  labels:
+    kubeaddons.mesosphere.io/name: azuredisk-csi-driver
+    kubeaddons.mesosphere.io/provides: csi-driver
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.5.0-1"
+    appversion.kubeaddons.mesosphere.io/azuredisk-csi-driver: "054.0"
```

Comment:
is this right? `054.0` or did you mean `0.54.0`

Reply:
yep, you are right this is typo 👍